Files/UserControls/Selection/RectangleSelection_DataGrid.cs

```diff
@@ -37,17 +37,28 @@ public RectangleSelection_DataGrid(DataGrid uiElement, Rectangle selectionRectan
 
         private void RectangleSelection_PointerMoved(object sender, PointerRoutedEventArgs e)
         {
+            if (scrollBar == null)
```

Comment:
More fancier way 😜 `if (scrollBar is not ScrollBar)`

Reply:
This definitely looks more philosophical this way :smile:
I preserved the original semantics (that were in line 48)

Reply:
@d2dyno1 how could you! you are missing the brackets, please follow Files code-style!! 😂

Reply:
NO_I_WILL_NOT